installer/Migration/V5_1_0/lang-string/performance.yaml

```diff
@@ -0,0 +1,46 @@
+langStrings:
+  -
+    value: 'Reviewers'
+    unitId: reviewers
+  -
+    value: 'Tracker Name'
+    unitId: tracker_name
+  -
+    value: 'Edit Performance Tracker'
+    unitId: edit_performance_tracker
+  -
+    value: 'Add Performance Tracker'
+    unitId: add_performance_tracker
+  -
+    value: 'Employee cannot be someone added as a reviewer'
```

Comment:
Not clear this phase

Reply:
Change to :-Employee cannot be assigned as his own reviewer

Reply:
Changed